### ISSUE TITLE
Roadmap 0.1

### ISSUE THREAD

**Issue by w41ter** [w41ter/urpc#1]
# Add and Finish Skeleton

- [x] #2
- [x] #3
- [x] #4
- [x] #5
- [x] #6
- [x] #7
- [x] #8
- [x] #9
- [x] #10
- [x] #11
- [ ] #12
- [x] #16
- [ ] #17
- [ ] #18

# Support Protocol

- [ ] #13
- [ ] #14
- [ ] #15

# More Efficient

- [ ] #19
- [ ] #20
- [ ] #21

# Integrate io_uring

TODO

**Comment by SGZW** [comment]
this rpc is design for io_uring?

**Comment by w41ter** [comment]
> this rpc is design for io_uring?

Yes, I want to try io uring, but I plan to use epoll first when designing the basic framework because I don't know the best practices for io uring yet.
